cmd/broker/bind_create_test.go

```diff
@@ -426,6 +450,43 @@ func TestCreateBindingEndpoint(t *testing.T) {
 	})
 }
 
+func assertResourcesExistence(t *testing.T, k8sClient client.Client, bindingID string) {
+	name := "kyma-binding-" + bindingID
+	namespace := "kyma-system"
+
+	serviceAccount := corev1.ServiceAccount{}
+	err := k8sClient.Get(context.Background(), client.ObjectKey{Name: name, Namespace: namespace}, &serviceAccount)
+	assert.NoError(t, err)
+	assert.NotNil(t, serviceAccount)
+
+	clusterRole := rbacv1.ClusterRole{}
+	err = k8sClient.Get(context.Background(), client.ObjectKey{Name: name, Namespace: namespace}, &clusterRole)
+	assert.NoError(t, err)
+	assert.NotNil(t, clusterRole)
+
+	clusterRoleBinding := rbacv1.ClusterRoleBinding{}
+	err = k8sClient.Get(context.Background(), client.ObjectKey{Name: name, Namespace: namespace}, &clusterRoleBinding)
+	assert.NoError(t, err)
+	assert.NotNil(t, clusterRoleBinding)
+}
+
+func assertServiceAccountsNotExists(t *testing.T, k8sClient client.Client, bindingID string) {
+	name := "kyma-binding-" + bindingID
```

Comment:
In `clean code world` it would call for dedicated function forming the name in one single place.

Reply:
Let me change it to function invocation instead of a method on one of bindings struct. It makes it easier to call such form from test because the tests operate on bindings structs that does not have id, but id is a part of binding name.

Reply:
Cool.